include/sound/sof/info.h

```diff
@@ -26,15 +26,6 @@
 #define SOF_IPC_INFO_LOCKSV		BIT(2)
 #define SOF_IPC_INFO_GDB		BIT(3)
 
-/* extended data types that can be appended onto end of sof_ipc_fw_ready */
-enum sof_ipc_ext_data {
-	SOF_IPC_EXT_UNUSED		= 0,
-	SOF_IPC_EXT_WINDOW		= 1,
-	SOF_IPC_EXT_CC_INFO		= 2,
-	SOF_IPC_EXT_PROBE_INFO		= 3,
-	SOF_IPC_EXT_USER_ABI_INFO	= 4,
-};
-
 /* FW version - SOF_IPC_GLB_VERSION */
```

Comment:
Shouldn't kernel be able to handle old fw, without extended manifest ?

Reply:
Good point. Extended FW boot data is enabled in up to SOF v1.6. I will test with v1.5 and v1.6 also.